form.tex

```diff
@@ -0,0 +1,90 @@
+\documentclass[a4paper]{article}
+
+\usepackage[bmargin=3cm, tmargin=2cm]{geometry}
+\usepackage{graphicx}
+\usepackage{xspace}
+\usepackage[latin1]{inputenc}
+\usepackage[pdftex]{hyperref}
+\setlength\parindent{0pt}
+
+\usepackage{helvet}
+\renewcommand*\familydefault{\sfdefault} %% Only if the base font of the document is to be sans serif
+\usepackage[T1]{fontenc}
+
+\pagenumbering{gobble}
+
+%%%% Defines for form filling
+\newcommand{\signature}{%
+    Signature:\\
+    \vspace{1cm}\\
+    \rule{7cm}{1pt}\\
+    Date: \vbox{\hbox{\TextField[bordercolor=, width=6cm]{\null}}\kern2pt\hrule}
+}
+
+\begin{document}
+
+\newcommand{\compyear}{2018\xspace}
+\newcommand{\competition}{SourceBots \compyear}
```

Comment:
As the rest of the media consent for is generic to an event, perhaps changing these variables to also just be `event` and `eventyear`? (I've looked, LaTeX doesnt have anything exceptionally nice for date formatting :'( )

Reply:
fixed in 916328f61419a2e66d9479a5bd9ba6b5f772e49b